process/sandbox.md

```diff
@@ -91,3 +91,16 @@ Frequency of reviews can be found on the [repo's README under "Frequency"](https
 ### Annual review
 
 Once in the Sandbox, projects are subject to an [Annual Review](https://github.com/cncf/toc/blob/master/process/sandbox-annual-review.md).
+
+### Sandbox departures
```

Comment:
Should this be called "sandbox archival" just to keep wording consistent and not introduce another term that needs to be understood by everyone?